src/model.c

```diff
@@ -192,6 +196,41 @@ void destroy_event_list( model *model, int type )
 	free( model->event_lists[type].events );
 };
 
+void set_up_default_occupation_network( model *model)
+{
+    model->occupation_network = calloc( N_DEFAULT_OCCUPATION_NETWORKS, sizeof( network* ) );
+    for( int idx = 0; idx < N_DEFAULT_OCCUPATION_NETWORKS; idx++ )
+        set_up_occupation_network( model, idx );
+}
+
+void set_up_custom_occupation_network( model *model )
```

Comment:
The guts of this function is almost a direct cut and paste of set_up_occupation_network, so we should not have both. 

Please can you change so that we have one version of this.

Reply:
Done. Also added code to set up default occupation networks with the same mechanism.